src/Core/MailTemplates/Handlebars/Layouts/TitleContactUs.html.hbs

```diff
@@ -0,0 +1,26 @@
+﻿{{#>FullHtmlLayout}}
+<div width="auto" cellpadding="0" cellspacing="0" style="padding:0; margin:-20px">
+    <div style="display: block; min-height: 204px; background-color: #175DDC; margin: 0;">
+        <div class="title-header-text" >
+            <div style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 24px; color: #ffffff; line-height: 32px; font-weight: 400; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none;">
+                {{TitleFirst}}<b class="white-title">{{TitleSecondBold}}</b>{{TitleThird}}
+            </div>
+        </div>
+        <div class="title-header-image" style="margin-left: auto; margin-right: 0px; vertical-align: bottom;">
+            <img style="margin-left: auto; margin-right: 0px; display: block;" alt='' src='https://assets.bitwarden.com/email/v1/business.png' />
+        </div>
+    </div>
+
+    {{>@partial-block}}
+
+    <div style="display:block;background-color: #FBFBFB;">
+        <div class="footer-text" style="display: inline-block; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; font-style: normal; font-weight: 400; font-size: 16px; line-height: 24px; padding-left: 35px; padding-right: 35px; margin-top: 15px; margin-bottom: 20px; margin-left: auto; margin-right: auto; vertical-align: middle; ">
+            <p style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; font-style: normal; font-weight: 600; font-size: 20px; line-height: 28px;">We’re here for you!</p>
+            If you have any questions, search the Bitwarden <a style="text-decoration: none; color: #175DDC; font-weight: 600;" href="https://bitwarden.com/help/">Help</a> site or <a style="text-decoration: none; color: #175DDC; font-weight: 600;" href="https://bitwarden.com/contact/">contact us</a>.
+        </div>
+        <div style="display: inline-block; width:33%; margin-left: auto; margin-right: auto; vertical-align: middle; ">
+            <img class="footer-image" src="https://assets.bitwarden.com/email/v1/chat.png" style="width: 94.73px; height: 77.25px; margin-left: auto; margin-right: 30px;" alt="" />
```

Comment:
Issue: Another alt description

Reply:
I added a comment to the PR saying that i would add the alt text but after reading a little about it, the best practice for decorative images it to leave them as empty.
Ref:
https://accessibility.huit.harvard.edu/describe-content-images
https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices